src/molecules/LockUps/TextD/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Div, P } from 'atoms/Elementals';
+
+const TextD = (props) => {
+  const { spacer, line1, line2, } = props;
+
+  return (
+    <Div spacer={spacer}>
+      <P spacer={1} typography={7} color='neutral-3'>{line1}</P>
+      <P>{line2}</P>
+    </Div>
+  );
+};
+
+TextD.propTypes = {
+  spacer: PropTypes.number,
+  line1: PropTypes.node,
+  line2: PropTypes.node,
+};
+
+TextD.defaultProps = {
+  spacer: 2,
+};
+
+export default TextD;
```

Comment:
It would be nice if we had a way of handling TextLockups similarly to the way we handle icons. Doing it this way could easily get unweildy and make it difficult to update / pivot our TextLockups in the future.